backend/build.gradle.kts

```diff
@@ -252,6 +254,56 @@ tasks {
             into("./LegalDocML.de")
         }
     }
+
+    register("processDBFixtures") {
+        var baseDir = fileTree("src/main/resources/db/data")
+        var norms =
+            baseDir.matching {
+                include("**/rechtsetzungsdokument-1.xml")
+            }
+
+        inputs.files(norms)
+
+        var outdir = layout.buildDirectory.dir("resources/main/db/data")
+        outputs.dir(outdir)
+
+        doLast {
+            norms.files.forEach { rechtsetzungsdokumentFile ->
+                var relativeNormFolder = rechtsetzungsdokumentFile.parentFile.relativeTo(baseDir.dir)
+                logger.info("Creating db migration for $relativeNormFolder")
+                var dbMigrationFile = file(outdir.get().file("$relativeNormFolder.sql"))
+
+                if (dbMigrationFile.exists()) {
```

Comment:
All this

```
if (dbMigrationFile.exists()) {
    dbMigrationFile.delete()
}
dbMigrationFile.parentFile.mkdirs()
dbMigrationFile.createNewFile()
```


Could just be replaced with:
```
dbMigrationFile.parentFile.mkdirs()
dbMigrationFile.writeText("")
```

Since it wil create the file if it doesnt exist and if it does it will just be ovewritten, meaning no need to manually delete/create files and this way reducing filesystem operatins